### PR TITLE
[xray] Adding tpl to customVolumeMounts

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:
[xray] Adding tpl to customVolumeMounts

### CODE REVIEW

File: stable/xray/templates/xray-statefulset.yaml [hunk 1]
```diff
@@ -500,12 +500,12 @@ spec:
...
-        {{- with .Values.common.customVolumeMounts }}
-{{ toYaml . | indent 8 }}
-        {{- end }}
-        {{- with .Values.persist.customVolumeMounts }}
-{{ toYaml . | indent 8 }}
-        {{- end }}
+{{- if .Values.common.customVolumeMounts }}
+{{ tpl .Values.common.customVolumeMounts . | indent 8 }}
+{{- end }}
+{{- if .Values.persist.customVolumeMounts }}
```
Comment (danielezer) [inline stable/xray/templates/xray-statefulset.yaml:506]: Please use with instead of the if statement

**Comment by evannorstrand** [comment]
Looks right to me. Has it been tested locally?

File: stable/xray/templates/xray-statefulset.yaml (hunk 1 above)
Comment (chukka) [reply stable/xray/templates/xray-statefulset.yaml]: @danielezer  Done

**Comment by chukka** [comment]
@evannorstrand We verified locally with customVolumes ,However can you please verify your scenario with  this PR and confirm. Once confimed, we will merge this PR

**Comment by evannorstrand** [comment]
Looks fine let’s get it merged

**Pull request merged**